Jenkinsfile

```diff
@@ -158,11 +158,22 @@ pipeline {
                 )
             }
         }
-        stage("Create a 'latest' tag from 'master'") {
+        stage("Create a 'dev' and 'latest' tag from 'master'") {
             when {
                 branch 'master'
             }
+            environment {
+                // qe-pipeline-library step
+                DOCKER_DEV_TAG = getDevTag()
```

Comment:
This will be available if [1] is merged.

[1] https://github.com/aerogear/qe-pipeline-library/pull/5

Reply:
already merged